candy.css

```diff
@@ -41,7 +41,6 @@ body {
     justify-content: space-evenly;
     align-items: center;
     width: 100%;
-    /* margin: 0 auto; */
```

Comment:
Comments: 
main -> that should be a class/if not needed - delete ,
`kreska ` - should be english name, 
`font-size `-  you have to choose one -> rem, em 
4vm => added a comment why you are using vh here

Reply:
Ah jesli chodzi o problem z navbarem -> moze zostac fixed ale musisz dodac ` overflow: hidden;` i pozniej dodac sobie `margin-top` do tej pierwszej sekcji :) powinno dzialac